examples/end-to-end-examples/finetuning/finetune_small_dataset.py

```diff
@@ -0,0 +1,52 @@
+
+import sys
+from enum import Enum
+from mcli.sdk import create_run, RunConfig
+
+# from config_registry import CONFIG_REGISTRY
+
+# User fills these in
+# TODO: make this work with argparse so the user doesn't have to modify the python file
+MODEL_NAME = '' # mpt-30b or mpt-7b
+TRAIN_DATA_PATH = '' # blobstore path to training data
+SAVE_FOLDER = '' # where to upload checkpoints
+
+class Models(Enum):
+  MPT_7B = 'mpt-7b'
+  MPT_30B = 'mpt-30b'
+
+
+def main(model_name: str, train_data_path: str, save_folder: str):
+    # Load the finetuning config
+    config: RunConfig = None
+    if model_name == Models.MPT_7B.value:
+        # TODO: Copy / move this YAML to the same folder as this python script
+        # Note that we use this one because it loads a model with pretrained: true
+        config = RunConfig.from_file('../sec_10k_qa/mcli-yamls/03_finetune_on_10ks.yaml')
```

Comment:
if you want to start with ift, this will need to be 04, but i would recommend defining your own config python dictionary, rather than importing from a different example.

Reply:
that way this example can have a "base" config dictionary, and then we can fill in the details (model name, save folder, etc)